spec/routing/cieszyn/welcome_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Cieszyn::WelcomeController, type: :routing do
   it 'routes to #show' do
-    path = 'cieszyn/welcome'
-    expect(get("/pl/#{path}")).to route_to("#{path}#show", locale: 'pl')
+    path = 'service'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.